gun-violence-dashboard/src/pages/Compare.js

```diff
@@ -1,5 +1,8 @@
 import React from "react";
 
+// use world ranking where possible
```

Comment:
Let's remove these comments

Reply:
I'm still working so I want these notes, they can be removed later